hrc/hrc/proto.hrc

```diff
@@ -125,6 +125,10 @@
     <location link="base/csharp.hrc"/>
     <filename>/\.cs$/i</filename>
   </prototype>
+  <prototype name="fsharp" group="main" description="F#">
```

Comment:
This is up to you and this is not really important.

What about forth.hrc, kotlin.hrc, fortran.hrc, etc?

F# is the member of .NET family, so it is not much different from C# or VB.NET. It is also a scripting language, so *scripts* is also a candidate category. But *rare* is a bit weird.

Personally, I think it is a strange idea to categorize schemes by popularity.